ui-ngx/src/app/modules/home/pages/dashboard/dashboard-form.component.ts

```diff
@@ -95,11 +95,6 @@ export class DashboardFormComponent extends EntityComponent<Dashboard> {
     this.entityForm.patchValue({configuration: {description: entity.configuration ? entity.configuration.description : ''}});
   }
 
-  prepareFormValue(formValue: any): any {
-    formValue.configuration = {...(this.entity.configuration || {}), ...(formValue.configuration || {})};
-    return formValue;
-  }
-
```

Comment:
This code broken saved dashboard. You need to use the parent class method prepareFormValue, before return value.